pkg/utils/validator/clusterresourceplacement_test.go

```diff
@@ -253,7 +253,7 @@ func Test_ValidateClusterResourcePlacement(t *testing.T) {
 	}
 }
 
-func Test_ValidateClusterResourcePlacement_RolloutStrategy(t *testing.T) {
+func TestValidateClusterResourcePlacementRolloutStrategy(t *testing.T) {
```

Comment:
```suggestion
func TestValidateClusterResourcePlacement_RolloutStrategy(t *testing.T) {
```

it's ok to use underscore to test one func but different scenarios.
The format is `Test{FuncName}_{scenario>}`